### ISSUE TITLE
touches.getAction is not a function

### ISSUE THREAD

**Issue by mars** [Popmotion/popmotion#194]
I just updated from my fork to the release with multitouch, version 7.7.0. This error is now coming from the new touches update function, [touches.js:72](https://github.com/Popmotion/popmotion/blob/master/src/actions/touches.js#L41):

```
[Error] TypeError: touches.getAction is not a function. (In 'touches.getAction(i)', 'touches.getAction' is undefined)
	updateActionWithTouches (commons.js:45673)
	updateTouches (commons.js:45637)
```

This code was working when I last tested it during my PR. Any idea what changed?

**Comment by InventingWithMonster** [comment]
Yep - it's 'getChildren' on release. This returns an array of actions which you can then do whatever with.

**Comment by InventingWithMonster** [comment]
I changed it to an array as providing an index was a bit opinionated. I also changed it to getChildren rather than getActions as it makes the relationship more explicit.

**Comment by mars** [comment]
In my opinion, it's a bit confusing that **Parallel** has **#addAction**, **#addActions**, and **#isActionComplete**, yet this particular interface has been changed from **#getAction** to **#getChildren**.

Anyway, the PR I just opened fixes it for me locally.

**Comment by mars** [comment]
> Popmotion adheres to [Semantic Versioning](http://semver.org/).

Does it? Changing existing API's so that cause breakage should increment the major version.

**Comment by InventingWithMonster** [comment]
Yeah that's a good point. I'm wondering if addChild would be more appropriate? I made this change originally because I agreed with your point that action.getActions() is a little weird and I thought getChildren would make more sense.

**Comment by mars** [comment]
Maybe add back **#getAction** to avoid breaking other people's code in a minor release, and then consider aliasing (or renaming with deprecation warnings) all of **Parallel**'s methods to "child/children" terminology.

* **#addAction** → **#addChild**
* **#addActions** → **#addChildren**
* **#isActionComplete** → **#areChildrenComplete**
* **#getAction** → **#getChild**

Thanks for the quick feedback today!

**Comment by InventingWithMonster** [comment]
No worries - I'm not in a position to publish tho

getAction was never actually published or documented. I think only you used
it pre-publish, I added it when I saw action.actions. If we got rid of
addActions that would be a breaking change though.

I agree with most of these except isActionComplete as technically it's
asking if parallel is complete. It's a detail that the criteria of that is
that all it's children are complete. Physics has different criteria but it
isn't reflected in the function name

**Comment by InventingWithMonster** [comment]
@mars this has been published now as 7.7.1

**Comment by mars** [comment]
Yay! I just verified that Touches function correctly in 7.7.1.  🎸🥓

So, finally deployed my project with official Popmotion instead of my fork. The [Portfolio section of this site](https://www.summerhall.design/portfolio) uses Touches **only on mobile** for swipe navigation, ignoring multi-touch gestures so that the browser's built-in zoom/pan may function correctly.
